src/components/feedback-message/FeedbackMessage.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { WarningIcon, InfoIcon } from '../icon';
+import { Tooltip, TooltipTrigger } from '../tooltip';
+import styles from './FeedbackMessage.css';
+
+const FeedbackMessage = ({ children, type, textColor, iconPosition, variant, tooltipChildren, className }) => {
+    const finalClassNames = classNames(
+        styles.feedbackMessage,
+        styles[type],
+        styles[iconPosition],
+        styles[variant],
+        className
+    );
+
+    const renderTooltip = () => {
```

Comment:
I would avoid creating functions inside the render method. Instead I would:

- Create `<FeedbackIcon>` and `<FeedbackTooltip>` components outside but in the same file and use them here
- Change the component to a class component and use render methods defined in it. This is related with a discussion we should have, as @paulobmarcos mentioned.